### PR TITLE
Limit log size to 500m

### PR DESCRIPTION
**Pull request by rfein-hearst**

### CODE REVIEW

**Comment by rfein-hearst** [comment]
Works as expected - relevant output of docker inspect

`      "LogConfig": {
        "Type": "json-file",
        "Config": {
          "max-size": "500m"
        }
`

**Comment by rfein-hearst** [comment]
thought about making this a tuneable but hopefully this should suffice for the remaining duration of our EOL deis infrastructure.

File: controller/Dockerfile [hunk 1]
```diff
@@ -3,9 +3,6 @@ FROM alpine:3.2
 # install common packages
 RUN apk add --update-cache curl bash sudo && rm -rf /var/cache/apk/*
 
-# install etcdctl
-RUN curl -sSL -o /usr/local/bin/etcdctl https://s3-us-west-2.amazonaws.com/get-deis/etcdctl-v0.4.9 \
-    && chmod +x /usr/local/bin/etcdctl
```
Comment (terowz) [inline controller/Dockerfile:6]: Probably check the router stuff assuming after the Microsoft buy that bucket is dead too?

https://github.com/Hearst-Hatchery/deis/blob/master/router/Dockerfile#L15

(I’ll test it Monday when we do the other change from Robert)

**Comment by rfein-hearst** [comment]
Ugh in controller they install it from the Github, why is it different in every component..

**Review by matthew-marchetti — APPROVED** [review]

**Comment by rfein-hearst** [comment]
don't think we need this after fixing sysdig spamming logs, so going to close unmerged.

**Pull request closed without merging**